src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -0,0 +1,332 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+import { MenuRegistry, MenuId } from 'vs/platform/actions/common/actions';
+import { ContextKeyAndExpr, ContextKeyEqualsExpr, ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
+import { KEYBINDING_CONTEXT_TERMINAL_PROCESS_SUPPORTED, TERMINAL_COMMAND_ID, TERMINAL_VIEW_ID } from 'vs/workbench/contrib/terminal/common/terminal';
+
+const enum ContextMenuGroup {
+	Create = '1_create',
+	Edit = '2_edit',
+	Clear = '3_clear',
+	Kill = '4_kill',
+	Config = '5_config'
+}
+
+export const enum ContextMenuTabsGroup {
```

Comment:
Since we're exporting this let's mention "Terminal", `TerminalTabContextMenuGroup`?